I'm getting errors when running `npm run test`:
```
➜  automaton-ts git:(AustinSickles/main) ✗ npm run test

> automaton-kit@1.0.0 pretest
> npm run compile


> automaton-kit@1.0.0 compile
> tsc

src/dfa/DFARunner.ts:225:41 - error TS2552: Cannot find name 'DFARunInstance_1'. Did you mean 'runInstance'?

225                 const newInstance = new DFARunInstance_1.default(transition.targetState, runInstance.remainingTokens.slice(1));
                                            ~~~~~~~~~~~~~~~~

  src/dfa/DFARunner.ts:207:19
    207             const runInstance = this.runInstances[runInstanceIdx];
                          ~~~~~~~~~~~
    'runInstance' is declared here.

src/dfa/DFARunner.ts:278:26 - error TS2531: Object is possibly 'null'.

278         returnarray.push(this.runInstances[0].currentState.label);
                             ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Found 2 errors in the same file, starting at: src/dfa/DFARunner.ts:225
```
Is there a particular reason why line 225 would need to use a different class `DFARunInstance_1` and constructor method than the regular `runStep()`?

--- a/src/dfa/DFARunner.ts
+++ b/src/dfa/DFARunner.ts
@@ -222,7 +222,10 @@ export default class DFARunner {
                 // state that this transition is targeting. Its input string will be
                 // whatever remains of the current input string after removing the
                 // first token in line.
-                const newInstance = new DFARunInstance_1.default(transition.targetState, runInstance.remainingTokens.slice(1));
+                const newInstance = new DFARunInstance(
+          		transition.targetState,
+          		runInstance.remainingTokens.slice(1)
+        	);
                 
                 newRunInstances.push(newInstance);
                 retLable=newInstance.currentState.label;

--- a/src/dfa/DFARunner.ts
+++ b/src/dfa/DFARunner.ts
@@ -228,7 +228,7 @@ export default class DFARunner {
         	);
                 
                 newRunInstances.push(newInstance);
-                retLable=newInstance.currentState.label;
+                retLable = newInstance.currentState?.label ?? "";
             }
             
         }
@@ -278,7 +278,7 @@ export default class DFARunner {
 
     runUntilConclusionDebug() {
         let returnarray = [];
-        returnarray.push(this.runInstances[0].currentState.label);
+        returnarray.push(this.runInstances[0].currentState?.label ?? "");
         while (!this.hasConcluded()) {
             returnarray.push(this.runStepDebug());
         }

--- a/src/dfa/DFARunner.ts
+++ b/src/dfa/DFARunner.ts
@@ -197,7 +197,7 @@ export default class DFARunner {
         // DFARunInstance objects. Each DFARunInstance contains a current position
         // in the automaton, and a remaining string of input tokens.
         
-        let newRunInstances = new Array();
+        let newRunInstances = new Array<DFARunInstance>();
         // Here, we iterate through all of the current run instance objects to
         // determine what the next set of run instance objects will be. Note that
         // a run instance object never lasts more than a single step - if it doesn't

